codegen/gateway.go

```diff
@@ -768,8 +769,19 @@ func NewGatewaySpec(
 	clientConfig string,
 	endpointConfig string,
 	middlewareConfig string,
+	copyrightHeaderFile string,
 	gatewayName string,
 ) (*GatewaySpec, error) {
+
+	copyright, err := ioutil.ReadFile(filepath.Join(
+		configDirName,
+		copyrightHeaderFile,
+	))
+	if err != nil {
```

Comment:
Let's make this optional, if it doesnt exist then copy right header is empty string.